Editor/Window/ConnectToFleetInput.cs

```diff
@@ -163,8 +167,10 @@ private async Task UpdateFleetMenu()
                     _fleetAttributes = new List<FleetAttributes>();
                 }
 
-                _fleetNameDropdownContainer.choices = _fleetAttributes.Select(fleet => fleet.Name).ToList();
-                _fleetNameDropdownContainer.value = _stateManager.AnywhereFleetName;
+                _fleetNameDropdownContainer.choices =
+                    _fleetAttributes.Select(fleet => $"{fleet.Name} | {fleet.FleetId}").ToList();
+                _fleetNameDropdownContainer.value =
+                    $"{_stateManager.AnywhereFleetName} | {_stateManager.AnywhereFleetId}";
```

Comment:
The text for the dropdown should be "fleetName | fleetId" , but the value itself should be "fleetId". This way when a selection is made you directly have the fleetId and don't need to split by the separator. This would be the right way since the value should be an identifier

Reply:
Whatever the value is is what is displayed, so this isnt really feasible.

Reply:
Ah damn, thanks for confirming